pallets/staking/src/lib.rs

```diff
@@ -1319,6 +1321,12 @@ decl_storage! {
         pub PermissionedValidators get(fn permissioned_validators):
             map hasher(twox_64_concat) T::AccountId => bool;
 
+        // Polymesh-Note: Polymesh specific changes to allow flexibility in commission.
+        /// Every validator has commission that should be in the range [0, Cap].
+        pub ValidatorCommissionCap get(fn validator_commission_cap) config(): Perbill;
+
+        // TODO: Need to be removed before mainnet launch.
+        // Keeping this to support the `on_runtime_upgrade`.
         /// Commission rate to be used by all validators.
         pub ValidatorCommission get(fn validator_commission) config(): Commission;
```

Comment:
We don't need to keep this. You can use `frame_support::migration::take_storage_value` to read the value without needing to define this.

Reply:
I thought it only works for the mapping